inference-engine/src/cldnn_engine/cldnn_remote_context.h

```diff
@@ -113,6 +113,7 @@ class typedCLDNNRemoteBlob : public TpublicAPI {
 protected:
     const std::shared_ptr<IAllocator> &getAllocator() const noexcept override { return _impl.getAllocator(); }
     void *getHandle() const noexcept override { return _impl.getHandle(); }
+    Blob* clone() const override { THROW_IE_EXCEPTION << "CLDNNRemoteBlobImpl contains unique ptr and is not copiable"; }
```

Comment:
If it is some member that prevents us from implementing `clone` method - is it reasonable to mention its name?
At the same, the fact, some derived classes does not provide full interface looks to me as a design flaw, but I guess it's unclear how to fix that...

Reply:
This class should obtain properly constructed cloned object. Now it's not in use, and have the stub only, but the real copy should be achievable.

Reply:
I still have question regarding mentioning member name in exception message

Reply:
that would be excessive. any refactoring involving rename might turn this message into incorrect. Too small detail to mention.